jinahub/segmenters/VADSpeechSegmenter/tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import os
+from typing import Callable
+from pathlib import Path
+
+from jina import Document, DocumentArray
+import numpy as np
+import torchaudio
+
+from ..vad_speech_segmenter import VADSpeechSegmenter
+
+
+@pytest.fixture(scope='module')
+def segmenter(tmpdir_factory) -> 'VADSpeechSegmenter':
+    workspace = tmpdir_factory.mktemp('data') / 'segmenter'
+    workspace.mkdir()
+    return VADSpeechSegmenter(
+        normalize=False, dump=True, metas={'workspace': str(workspace)}
+    )
+
+
+@pytest.fixture
+def test_dir() -> str:
+    return os.path.dirname(os.path.abspath(__file__))
```

Comment:
Not needed, replace with `Path(__file__).parent` where used

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/97/commits/26ae02fc9d1b12c0c381f0495f4f1ae1d900b677) for the change.